library/manageiq_provider.py

```diff
@@ -75,6 +75,18 @@
       - the provider api auth token
     required: true
     default: null
+  provider_verify_ssl:
+    description:
+      - whether SSL certificates should be verified for HTTPS requests between
+      ManageIQ and the provider
+    required: false
```

Comment:
I think I'll keep it, for now at least, to maintain consistency and give a little more info to the users.